AEPIdentity/Tests/IdentityFunctionalTests.swift

```diff
@@ -383,18 +388,16 @@ class IdentityFunctionalTests: XCTestCase {
         XCTAssertEqual(EventSource.responseIdentity, dispatchedEvent?.source)
         XCTAssertNotNil(dispatchedEvent?.data?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID])
         let ecid = dispatchedEvent?.data?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID] as? String ?? ""
-        XCTAssertFalse(ecid.isEmpty);
+        XCTAssertFalse(ecid.isEmpty)
     }
 
-    /// Tests that getExperienceCloudId doesnt wait for Identity bootup and ECID to be generated when cached ECID is not avaiable
-    func testGetECIDWaitsForConfigurationSharedStateToResolveWhenCachedECIDNotAvailable() {
+    func testGetECIDAfterForceSyncDoesNotWaitForConfigurationSharedState() {
```

Comment:
for this test we need to set the shared state on pending after force sync:
1. set valid config
2. dispatch any event -> initiates force sync
3. set config nil data and .pending
4. dispatch get ecid event
5. assert returns data